src/com/company/ppj08/Tasks45.java

```diff
@@ -0,0 +1,46 @@
+package com.company.ppj08;
+
+public class Tasks45 {
+    public static void main(String[] args) {
+        task4();
+        task5();
+    }
+
+    private static void task4() {
+        int day = 4;
+        int month = 4;
+        int countDays = 0;
+        for (int numberMonth = 1; numberMonth < month; numberMonth++) {
+            countDays += 31;
+            if (numberMonth == 2) {
```

Comment:
Why we check that numberMonth equals only `2`, the program should return correct result for any data in any month during the ear. Please finish the solution.

Reply:
fixed